### DISCUSSION TITLE
🖋: React Native公式のGetting Startedで必要な設定を読み飛ばしてしまう

### DISCUSSION THREAD

**Discussion by takei0w0** [ws-4020/mobile-app-crib-notes#125] [Ideas]
学習コンテンツ記載内容と重複している箇所があるため、重複箇所は読み飛ばして設定する必要があるが、必要な箇所まで読み飛ばしてしまう人がいた。

必要な箇所がわかりにくいのが問題？

**Comment by creature-water-valley** [comment]
現在は、[開発環境の構築](https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/getting-started/setting-up-development-environment)のみを参照することで環境構築ができるようになっているため、クローズ
